net/v2ray/Portfile

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+name                v2ray
+version             4.21.3
+categories          net security
+platforms           darwin
+license             MIT
+maintainers         {@casouri gmail.com:casouri} openmaintainer
+description         A platform for building proxies to bypass network restrictions
+
+long_description    Project V is a set of network tools that help you to build your own computer network. \
+                    It secures your network connections and thus protects your privacy.
+
+homepage            https://www.v2ray.com
+go.setup            github.com/v2ray/v2ray-core ${version} v
+
+checksums           rmd160  99206bc61254defe0dbec2e45deabf458dfaa647 \
+                    sha256  c0babc58c299483107c8ed00b5834962fef6f56a3f0b38835ef70cd7f42e6026 \
+                    size    6828463
+
+use_configure       no
+
+build {
+    system -W ${worksrcpath}/main "env CGO_ENABLED=0 go build -o ./v2ray -ldflags \"-s -w\""
+    system -W ${worksrcpath}/infra/control/main "env CGO_ENABLED=0 go build -o ./v2ctl -tags confonly -ldflags \"-s -w\""
+}
```

Comment:
I was under the impression that the go portgroup handled some of this? (Note that I know far too little about the portgroup; I may be completely wrong.)

Reply:
Let me have a second look at the port group. My memory is a bit vague now...

Reply:
According to the manual "By default this PortGroup runs go build from the ${worksrcpath}. Assuming this results in a binary with the same name as the project, and that there are no other files to install". As you can see, this port is a bit more complicated than that.

Reply:
Yah, but there are no other variables you can set to customize the build environment and flags? You're sure of that?

Reply:
Hey I asked around on irc and the dudes there said there aren’t much that I can do. Because I have to compile in two different locations.